### PR TITLE
Adding compas_cgal

### PR DESCRIPTION
**Pull request by brgcode**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by brgcode** [comment]
@gonzalocasas

**Comment by brgcode** [comment]
@conda-forge/help-python This package is ready for review.

**Review by gonzalocasas — APPROVED** [review]
LGTM!

**Comment by gonzalocasas** [comment]
This is ready for review.
@conda-forge-admin, please ping team

File: recipes/compas_cgal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/compas-dev/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  skip: true  # [py<36]
+
```
Comment (ocefpaf) [inline recipes/compas_cgal/meta.yaml:16]: You can remove the platform depended build scrips and add `script: {{ PYTHON }} -m pip install . --no-deps -vv` here.

File: recipes/compas_cgal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - eigen
+  host:
+    - pip
+    - python
+    - pybind11
+    - cgal-cpp >=5.0
+  run:
+    - python {{ python }}
```
Comment (ocefpaf) [inline recipes/compas_cgal/meta.yaml:28]: You don't need that jinja variable for pyhton.

File: recipes/compas_cgal/meta.yaml (hunk 1 above)
Comment (brgcode) [reply recipes/compas_cgal/meta.yaml:17]: done

File: recipes/compas_cgal/meta.yaml (hunk 2 above)
Comment (brgcode) [reply recipes/compas_cgal/meta.yaml]: done

**Comment by brgcode** [comment]
@ocefpaf successful build of updated recipe...

**Pull request merged**